src/CodeGen_OpenCL_Dev.cpp

```diff
@@ -413,15 +429,23 @@ void CodeGen_OpenCL_Dev::CodeGen_OpenCL_C::visit(const Store *op) {
     Expr ramp_base = strided_ramp_base(op->index);
     if (ramp_base.defined()) {
         internal_assert(op->value.type().is_vector());
-        string id_ramp_base = print_expr(ramp_base);
 
-        stream << get_indent() << "vstore" << t.lanes() << "("
-               << id_value << ","
-               << 0 << ", (" << get_memory_space(op->name) << " "
-               << print_type(t.element_of()) << "*)"
-               << print_name(op->name) << " + " << id_ramp_base
-               << ");\n";
+        if((op->alignment.modulus % op->value.type().lanes() == 0)
```

Comment:
space between `if` and `(` in Halide code style

Reply:
modulus/modulus check again

Reply:
Done.